server/handler/options.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2022 Palantir Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handler
+
+import (
+	"github.com/palantir/bulldozer/bulldozer"
+	"os"
+	"strconv"
+)
+
+const (
+	DefaultSharedRepository = ".github"
+	// DefaultConfigurationPath
+	// The default configuration path is the same for the repo as
+	// for the shared config if not configured
+	DefaultConfigurationPath = "bulldozer.yml"
```

Comment:
There's actually a subtle different here. To match the docs, the example configuration, and the (I think) expected behavior, the default configuration path should be the hidden file `.bulldozer.yml`, while the default shared configuration path should be the regular file `bulldozer.yml`. 

This difference allows the shared configuration repository to define its own Bulldozer options that may be different from the configuration used by other repositories in the organization.

Reply:
Yeah thank you for pointing that out, I must have missed that.